src/plugins/projectexplorer/projectmodels.cpp

```diff
@@ -205,15 +207,16 @@ FlatModel::FlatModel(QObject *parent)
 
 QVariant FlatModel::data(const QModelIndex &index, int role) const
 {
-    const Node * const node = nodeForIndex(index);
+    const Node *const node = nodeForIndex(index);
     if (!node)
         return QVariant();
 
-    const FolderNode * const folderNode = node->asFolderNode();
-    const ContainerNode * const containerNode = node->asContainerNode();
-    const Project * const project = containerNode ? containerNode->project() : nullptr;
-    const Target * const target = project ? project->activeTarget() : nullptr;
-    const BuildSystem * const bs = target ? target->buildSystem() : nullptr;
+    const FolderNode *const folderNode = node->asFolderNode();
+    const ContainerNode *const containerNode = node->asContainerNode();
+    const Project *const project = containerNode ? containerNode->project() : nullptr;
+    const Target *const target = project ? project->activeTarget() : nullptr;
+    const BuildSystem *const bs = target ? target->buildSystem() : nullptr;
```

Comment:
This seems to be a code formatting change. We should avoid those to reduce risk of conflicts here.